google-datacatalog-sisense-connector/src/google/datacatalog_connectors/sisense/prepare/datacatalog_entry_factory.py

```diff
@@ -81,8 +81,32 @@ def make_entry_for_dashboard(
             update_timestamp.FromDatetime(modified_datetime)
             entry.source_system_timestamps.update_time = update_timestamp
 
+        entry.schema = self.__make_schema_for_dashboard(dashboard_metadata)
+
         return generated_id, entry
 
+    @classmethod
+    def __make_schema_for_dashboard(
+            cls, dashboard_metadata: Dict[str, Any]) -> Optional[Schema]:
+
+        if not dashboard_metadata.get('filters'):
+            return
+
+        filters_column = datacatalog.ColumnSchema()
+        filters_column.column = 'filters'
+        filters_column.type = 'array of filters'
```

Comment:
Type is an open field, but is there any benefit on specifying it as `array of filters`?

With the column name it's clear that you have filters there... unless I am missing something, I would suggest just sticking with standard types, maybe just `array`?

Reply:
I'm ok with `array`. Thanks!